modules/network-observability-configuring-options-overview.adoc

```diff
@@ -7,7 +7,7 @@
 = Configuring advanced options for the Overview view
 You can customize the graphical view by using advanced options. To access the advanced options, click *Show advanced options*. You can configure the details in the graph by using the *Display options* drop-down menu. The options available are:
 
-* *Scope*: Select the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, service, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
+* *Scope*: Select the detail of components between which the network traffic flows. You can set the scope to *Node*, *Namespace*, *Owner*, or *Zones*, *Resource*. *Owner* is an aggregation of resources. *Resource* can be a pod, service, node, in case of host-network traffic, or an unknown IP address. The default value is *Namespace*.
```

Comment:
Thanks @jpinsonneau . In the Sprint demo on Thursday, I realized there was this configuration piece. I'm sorta at a point where I'm thinking we may need to have a small procedure about configuring these multi-cluster options, even if mutli-cluster isn't a splashy feature we're promoting right now. I feel like unless I mention multi-cluster, adding this configuration just doesn't make sense. Our only other option if we don't want to mention multi-cluster right now is to just add this mention of zones in the release note like we did for cluster ID last release.

Reply:
Since it's just 2 flags we could introduce these in the [Installing the Network Observability Operator
](https://docs.openshift.com/container-platform/4.14/network_observability/installing-operators.html#network-observability-operator-installation_network_observability) section. We gained a lot of room with easier loki config.

WDYT ?

Reply:
Else, similar to conversation tracking, packet drops & DNS, we can add subtitles under [Observing the network traffic from the Traffic flows view](https://docs.openshift.com/container-platform/4.14/network_observability/observing-network-traffic.html#network-observability-trafficflow_nw-observe-network-traffic) for multi cluster and zones.

That's probably better for future improvments since each feature is supposed to grow

Reply:
I think I'm leaning towards adding a configuration procedure similar to the ones you list in Observing the network traffic from the Traffic flows view. I agree, I think doing it this way will lend better to future growth of  the multi-cluster feature.